formats/definitions.json

```diff
@@ -20,6 +20,26 @@
       ],
       "description": "A short identifier we send to Google Analytics for multi-valued fields. This means we avoid the truncated values we would get if we sent the path or slug of eg organisations."
     },
+    "external_related_links":{
```

Comment:
If you pull from master and run the default task the whitespace issue above should be fixed automatically.